jetty-server/src/main/config/modules/well-known.mod

```diff
@@ -0,0 +1,22 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Serve static files from a directory for the "/.well-known" context path.
+
+[tags]
+handler
+
+[depend]
+server
+
+[xml]
+etc/well-known.xml
+
+[files]
+.well-known/
+
+[ini-template]
+# tag::documentation[]
+## Well Known Directory (relative to $jetty.base)
```

Comment:
Please specify here that if it's a relative path then it is relative to `$JETTY_BASE`, otherwise it's an absolute path.